test/jruby/test_sprintf.rb

```diff
@@ -0,0 +1,19 @@
+require 'test/unit'
```

Comment:
a reference should be added to https://github.com/jruby/jruby/blob/master/test/jruby.index